Documentation/network/ebpf/maps.rst

```diff
@@ -93,3 +93,25 @@ their own connection tracking tables when Cilium is configured with
 +------+--------------+-----------------------+-------------------+
 |   96 |          360 |               3145728 |           4552960 |
 +------+--------------+-----------------------+-------------------+
+
+.. _svc_lb_tuning:
+
+Service LB Map Tuning
+=====================
+
+Cilium uses the LB services maps named ``cilium_lb{4,6}_services_v2`` to hold Service load balancer entries for clusterIP and nodePort service types.
+These maps are configured via the ``--bpf-lb-map-max`` flag and are set to 64k by default. If this map is full, Cilium may be unable to reconcile Service
+updates which may affect connectivity to service IPs or the ability to create new services.
+
+The required size of service LB maps depends on multiple factors. Each clusterIP/nodePort service will create
+a number of entries proportional to the number of backends per service (i.e. Pods selected by the Service>)
+times the number of port/protocol entries in the respective Service spec.
+
+Using this, we can estimate the required map size as:
```

Comment:
> Each clusterIP/nodePort service creates a number of entries proportional to the number of [...] times the number of [...].

I don't think each service creates a number “proportional” to the result from the multiplication, it just creates the resulting number of entries? (1 service -> nb of endpoints x port/protocols; but _not_ a multiple of this? Not counting x1 as a relevant factor)

```suggestion
The required size of service LB maps depends on multiple factors. Each clusterIP/nodePort service creates
a number of entries equal to the number of backends per service (i.e. Pods selected by the Service),
multiplied by the number of port/protocol entries in the respective Service specification. As a result, we can estimate the required map size as:
```

Reply:
Yeah correct, I'll make this more precise 👍